wagtail/admin/templates/wagtailadmin/shared/breadcrumbs.html

```diff
@@ -20,6 +20,7 @@
              data-w-breadcrumbs-open-icon-class="icon-breadcrumb-expand"
              data-w-breadcrumbs-opened-content-class="w-max-w-4xl"
              data-w-breadcrumbs-peek-target-value="header"
+             data-w-breadcrumbs-store-value="header"
```

Comment:
```suggestion
             data-w-breadcrumbs-storage-key-value="header"
```

As per above comments, let's use a name that's a bit clearer to understand.